src/util.ts

```diff
@@ -24,7 +24,8 @@ export async function setViewport(
     fitWindow: false, // as we cannot resize the window, `fitWindow: false` is needed in order for the viewport to be resizable
   }
 
-  const versionResult = await CDP.Version()
+  const { host, port } = client
```

Comment:
Have you tested this? client is neither an object, nor does it contain host and port. You probably need to send the options as an arg to the function.